CMSgov/mint-app
# [EASI-2521] Discussion and Discussion Reply return user info full stack

# EASI-2521

## Changes and Description

- Added UserInfo to Discussion
- Added UserInfo to DiscussionReply



## How to test this change

- Create a Discussion, observe that CreatedByUserInfo populates appropriately and ModifiedByUserInfo is nil
- Update a Discussion, observe that CreatedByUserInfo populates without change and ModifiedByUserInfo populates appropriately
- Create a DiscussionReply, observe that CreatedByUserInfo populates appropriately and ModifiedByUserInfo is nil
- Update a DiscussionReply, observe that CreatedByUserInfo populatese without change and ModifiedByUserInfo populates appropriately

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
